docs/products/kafka/kafka-connect/howto/debezium-source-connector-pg.rst

```diff
@@ -1,7 +1,7 @@
-Create a Debezium source connector for PostgreSQL
-=================================================
+Create a Debezium source connector for PostgreSQL®
```

Comment:
What about Debezium? Do we want to tackle it in another PR?

Reply:
The Debezium website doesn't ask for any use of ® or ™, so I believe we are OK with this as it is.